src/services/token.service.js

```diff
@@ -0,0 +1,24 @@
+const { Token } = require('../models/token.model');
+
+const getByToken = async (refreshToken) => {
+  return Token.findOne({ where: { refreshToken } });
+};
+
+const save = async (userId, newToken) => {
+  const token = await Token.findOne({ where: { userId } });
+
+  if (!token) {
+    await Token.create({ userId, refreshToken: newToken });
+
+    return;
+  }
+
+  token.refreshToken = newToken;
+  await token.save();
```

Comment:
Consider adding error handling for the `token.save()` operation to manage potential errors and prevent unhandled promise rejections.